lib/mappers/scdl_mapper.py

```diff
@@ -21,13 +21,10 @@ def map_data_provider(self):
 
     def map_collection(self):
         """//dct:isPartOf -> sourceResource.collection"""
-        collection = None
-
         if exists(self.provider_data, 'isPartOf'):
-            for coll in iterify(getprop(self.provider_data, 'isPartOf')):
-                collection = {"collection": {"title": coll}}
-        if collection:
-            self.update_source_resource(collection)
+            self.update_source_resource(
+                {'collection': iterify(getprop(self.provider_data,
```

Comment:
This mapping should be to the title child prop 
`{'collection': {'title': ____ } }`